src/commands/oftenAsked/ping.ts

```diff
@@ -0,0 +1,12 @@
+import { create_command } from '../../command';
+
+export const command = create_command(
```

Comment:
use https://github.com/lenscas/thanks_bot_2/blob/ed83f073d020f7d663c70fc60f1fe8d511c457ff/src/command.ts#L47 instead so it is limited to the command channel, rather than being available everywhere.

It also should go in the `bot` group. As it is related to the bot, rather than giving an answer to an often asked question

Reply:
Oh okay. Will fix :+1: